tests/Get-DbaDbCompression.Tests.ps1

```diff
@@ -32,8 +32,10 @@ Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
     Context "Command handles heaps and clustered indexes" {
         It "Gets results" {
             $results | Should Not Be $null
+            $results.Database | Get-Unique | Should -Be $dbname
+            $results.DatabaseId | Get-Unique | Should -Be $server.Query("SELECT database_id FROM sys.databases WHERE name = '$dbname'").database_id
```

Comment:
Believe this only works because we are still on Pester v4 in v5 it won't execute script commands found in the It block and would always fail. Do not have to change now but we will once we start rewriting tests for v5 Pester...if we ever get around to it.